cypress/e2e/common/admin.js

```diff
@@ -95,3 +95,19 @@ Then(
     cy.get('div.notice > i.icon.icon-important').should('not.exist');
   },
 );
+
+Then(
+  'long, long feedback details from user {string} displays correctly in the page',
```

Comment:
Very nit-picky I know, but could "long, long" be renamed to something like "very long"? Just because it might choke on the comma at some point. Murphy's law ......